components/TextFields/src/MDCTextInputControllerLegacyDefault.m

```diff
@@ -317,6 +337,8 @@ - (void)commonMDCTextInputControllerLegacyDefaultInitialization {
   _disabledColor = [self class].disabledColorDefault;
   _floatingEnabled = [self class].isFloatingEnabledDefault;
   _internalCharacterCounter = [[MDCTextInputAllCharactersCounter alloc] init];
+  _underlineHeightActive = [self class].underlineHeightActiveDefault;
```

Comment:
Based on the "full width" controller below, this looks like an override point for subclasses. Please consider moving it to a public (or private) header, or maybe document that it's only for first-party subclasses.